### PR TITLE
namshi/jose none header fix

### PR DESCRIPTION
**Pull request by cirpo**

### CODE REVIEW

**Comment by cirpo** [comment]
@odino @geshan @p16 blease jack

File: source/_posts/2015-06-16-jose-fix.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+title: "Important 'namshi/jose' jws library fix"
+date: 2015-06-16 16:39
+comments: true
+categories: namshi/jose, jws, opensource, 
+author: Alessandro Cinelli (cirpo)
+---
+
+>We just released a very important security fix on **namshi/jose**, our opensource lightweight implementation of the JWS (JSON Web Signature) in PHP.
+
+The issue is related to the fact that any jws token was valid if the algorithm specified in the header was 'none' (first 'n' lower case).
```
Comment (p16) [inline source/_posts/2015-06-16-jose-fix.markdown]: wasn't it because the "n" was uppercase? ("N")

I may have got that wrong :)

**Comment by geshan** [comment]
looks ok to me @cirpo

**Comment by p16** [comment]
:+1:

**Pull request merged**